storefront/features/Header/Header.tsx

```diff
@@ -5,7 +5,7 @@ import Navbar from "../../containers/Navbar/Navbar";
 const Header = () => {
   return (
     <Wrapper>
-      <Heading textAlign="center">Aixmazone</Heading>
+       <img src="/aixmazon.png" alt="Aixmazon" width={189} height={60} />
```

Comment:
:+1: for specifying dimensions. 

Regarding accessibility, can't you add something that could help blind people that can't see images for example?

Reply:
@On1zuma could you please close #13 and instead add a new commit into this branch & push it? You can probably event do something like that:
```
git fetch
git switch -c feat-add-logo-aixmazon origin/feat-add-logo-aixmazon
git cherry-pick 5aaf624fee8e5755d0cf8cebd45c18df71a4e74a feabee49798321c5109512834fd09fa59e156f4c
git rebase origin/main
git push -f
```

Reply:
I've seen #13 
No need to make another PR unless I merge this one. Ok so as I merge this one right now you will need to make another PR. But you can checkout anytime a branch and add some commits